iulianIachimov/src/main/java/datacollectiontask/additional_modifications/New_Main.java

```diff
@@ -0,0 +1,23 @@
+package datacollectiontask.additional_modifications;
+
+public class New_Main {
+     /*
```

Comment:
please get rid of commented code, all classes

Reply:
Referring to all classes - should there be no comment lines at all?

Reply:
you could keep comments as a Java doc, but no commented code

Reply:
Thank you for the tips, the changes were done.